runtime/gc_glue_java/ScavengerDelegate.cpp

```diff
@@ -352,10 +352,14 @@ MM_ScavengerDelegate::doContinuationSlot(MM_EnvironmentStandard *env, omrobjectp
 #endif /* JAVA_SPEC_VERSION >= 24 */
 
 void
-MM_ScavengerDelegate::doStackSlot(MM_EnvironmentStandard *env, omrobjectptr_t *slotPtr, MM_ScavengeScanReason reason, bool *shouldRemember, void *walkState, const void* stackLocation)
+MM_ScavengerDelegate::doStackSlot(MM_EnvironmentStandard *env, omrobjectptr_t *slotPtr, MM_ScavengeScanReason reason, bool *shouldRemember, J9Object *fromObject, void *walkState, const void *stackLocation)
```

Comment:
Why are you passing fromObject - does not seem to be used?
i understand it was needed in CopyForward to determine the Allocation Context.

Reply:
removed it